projects/core/src/util/script-loader.service.ts

```diff
@@ -39,6 +40,7 @@ export class ScriptLoader {
     callback?: EventListener;
     errorCallback?: EventListener;
     placement?: ScriptPlacement;
+    disableKeyRestriction?: boolean;
```

Comment:
Not a breaking change as adding optional property.
No impact on current behavior when this property is undefined.